code/game/gamemodes/clock_cult/clock_scriptures/scripture_applications.dm

```diff
@@ -236,15 +171,15 @@
 	quickbind_desc = "Creates a Mania Motor, which causes minor damage and negative mental effects in non-Servants."
 
 
-//Tinkerer's Daemon: Creates an efficient machine that rapidly produces components at a power cost.
+/*//Tinkerer's Daemon: Creates an efficient machine that rapidly produces components at a power cost.
```

Comment:
I assume these comments are merely part of the WIP nature.
if they are not, then remove them, Git does a better job than us at maintaining history.